apps/server-asset-sg/src/run-command.ts

```diff
@@ -0,0 +1,26 @@
+import 'reflect-metadata';
+
+import { exit } from 'process';
+import { CommandFactory } from 'nest-commander';
+import { AppLogger } from '@/app.logger';
+import { AppModule } from '@/app.module';
+
+async function bootstrap() {
```

Comment:
As discussed today, we should add an `oh snap` early return if we - by whatever ungodly accidents - were to run this in dev, int or prod environments.